js/main.js

```diff
@@ -32,6 +36,92 @@ function entryFormData(event) {
   data.entries.unshift(formInputs);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+
+  var li = document.createElement('li');
+  li.setAttribute('class', 'row margin-bottom-li');
+
+  var imgDiv = document.createElement('div');
+  imgDiv.setAttribute('class', 'column-half');
+  var imgView = document.createElement('img');
+  imgView.setAttribute('class', 'cover img-container-view');
+  imgView.setAttribute('src', data.entries[0].imgURL);
+  imgView.setAttribute('alt', 'journal-entry-image');
+
+  var textDiv = document.createElement('div');
+  textDiv.setAttribute('class', 'column-half');
+  var h2Title = document.createElement('h2');
+  h2Title.textContent = data.entries[0].title;
+  var pNotes = document.createElement('p');
+  pNotes.textContent = data.entries[0].notes;
+
+  imgDiv.appendChild(imgView);
+  textDiv.appendChild(h2Title);
+  textDiv.appendChild(pNotes);
+  li.appendChild(imgDiv);
+  li.appendChild(textDiv);
+  ul.prepend(li);
```

Comment:
You are repeating this behavior multiple times, this should be its own function that receives a single entry object and `return`s the DOM tree for that entry. You should not append it to the page inside of that function, you should append the return value of that function where the function is called. I encourage you to go back to `dom-creation` to review the exercise and the code you wrote for it to get a refresher.